pulsar-broker/pom.xml

```diff
@@ -335,6 +356,12 @@
       <type>pom</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.jclouds</groupId>
+      <artifactId>jclouds-allblobstore</artifactId>
+      <version>${jclouds.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Same as above.

Reply:
I think this is needed, we have spent some time discussing dependence.